system-parachains/bridge-hubs/bridge-hub-kusama/src/bridge_to_polkadot_config.rs

```diff
@@ -163,11 +163,19 @@ pub type BridgeGrandpaPolkadotInstance = pallet_bridge_grandpa::Instance1;
 impl pallet_bridge_grandpa::Config<BridgeGrandpaPolkadotInstance> for Runtime {
 	type RuntimeEvent = RuntimeEvent;
 	type BridgedChain = bp_polkadot::Polkadot;
-	type MaxFreeMandatoryHeadersPerBlock = ConstU32<4>;
 	type HeadersToKeep = RelayChainHeadersToKeep;
+	type MaxFreeHeadersPerBlock = ConstU32<4>; // FAIL-CI @svyatonik
+	type FreeHeadersInterval = ConstU32<5>; // FAIL-CI @svyatonik
 	type WeightInfo = weights::pallet_bridge_grandpa::WeightInfo<Runtime>;
 }
 
+// FAIL-CI @svyatonik please check this.
+impl pallet_bridge_grandpa::WeightInfoExt for weights::pallet_bridge_grandpa::WeightInfo<Runtime> {
+	fn submit_finality_proof_overhead_from_runtime() -> Weight {
+		todo!()
```

Comment:
@svyatonik there were lots of changes to the bridges pallets, please check all `FAIL-CI @svyatonik` strings in the code and fix them.  
Code should compile, but i had to put `todo!()` to make it compile.

Reply:
@ggwpez I will take care of `FAIL-CI @svyatonik`

Reply:
fixed here: https://github.com/polkadot-fellows/runtimes/pull/352